packages/staking/src/features/drawer/StakePoolDetails.tsx

```diff
@@ -34,7 +34,7 @@ type DraftPortfolioInvalidReason = 'invalid-allocation' | 'slider-zero';
 type DraftPortfolioValidity = { valid: true } | { valid: false; reason: DraftPortfolioInvalidReason };
 
 const getDraftPortfolioValidity = (store: DelegationPortfolioStore): DraftPortfolioValidity => {
-  if (!store.draftPortfolio || store.draftPortfolio.length === 0) return { valid: true }; // throw new Error('Draft portfolio is not defined');
+  if (!store.draftPortfolio || store.draftPortfolio.length === 0) return { valid: true };
```

Comment:
nitpick - I'd consider re-writing it a bit more concisely:

```suggestion
  if (!store.draftPortfolio?.length) return { valid: true };
```

Reply:
I think our lint rules will allow for
```suggestion
  if (store.draftPortfolio?.length !== 0) return { valid: true };
```